src/test/java/de/tum/in/www1/artemis/TextAssessmentIntegrationTest.java

```diff
@@ -160,4 +159,17 @@ public void getResultWithPredefinedTextblocks_studentHidden() throws Exception {
         assertThat(result).as("saved result found").isNotNull();
         assertThat(result.getParticipation().getStudent()).as("student of participation is hidden").isNull();
     }
+
+    @Test
+    @WithMockUser(value = "tutor1", roles = "TA")
```

Comment:
I probably get it wrong, but the assessor should not be visible to the student. Why did you define `TA` (teaching assistant) as role here?

Reply:
@sascha11110 No, you were completely right! I fixed it and added an additional test, can you re-check it please? :)